internal/core/entities/game_room/game_room.go

```diff
@@ -130,6 +130,7 @@ type GameRoom struct {
 	IsValidationRoom bool
 	LastPingAt       time.Time
 	CreatedAt        time.Time
+	OccupiedSlots    int
```

Comment:
nit: why don't we use the same naming as the proto? IMO it would simplify maintenance and they seem to map 1:1, right?